testing/src/main/scala/sbt/JUnitXmlTestsListener.scala

```diff
@@ -59,7 +61,7 @@ class JUnitXmlTestsListener(val outputDir: String) extends TestsListener {
    * Gathers data for one Test Suite. We map test groups to TestSuites.
    * Each TestSuite gets its own output file.
    */
-  class TestSuite(val name: String) {
+  class TestSuite(val name: String, timestamp: Long) {
```

Comment:
I'd suggest using `java.imte.LocalDateTime` here.

Reply:
Ahhh I avoided this because I didn’t know if sbt supports jdk 7 and older.

Reply:
Ahhh I avoided this because I didn’t know if sbt supports jdk 7 and older.

Reply:
sbt 1 is on Scala 2.12 so we can assume JDK 8 and up.